packages/slice-machine/src/modules/availableCustomTypes/index.ts

```diff
@@ -114,13 +122,49 @@ export const availableCustomTypesReducer: Reducer<
         ...normalizedNewCustomType,
       };
     }
+
+    case getType(saveCustomTypeCreator.success): {
+      if (!state) return state;
+      const customTypeId = action.payload.customTypeId;
```

Comment:
think we should pass to the payload the actual `customType`, not just its id. Since the CT we are saving is now different than the outdated one in the availableCustomType.

so where you have the `saveCustomTypeCreator.success({ customTypeId: currentCustomType.id })` you can do `saveCustomTypeCreator.success({ customType: currentCustomType})`, and then you can use this payload to just set it